.github/workflows/scripts/issues/addClosingMessage.js

```diff
@@ -41,8 +41,13 @@ module.exports = async ({ core, context, github }) => {
 
     core.info(`>>> Author permission level: ${userPermission.data.permission}`);
 
+    // checks if the issue was created from a customer with commercial support (pro, premium or priority plan)
+    const isPaidSupport = issue.labels.some((label) =>
+      /\b(pro|premium|priority)\b/i.test(label.name),
+    );
+
     // Only ask for feedback if the user is not an admin or has at least write access (from a team membership)
```

Comment:
the grammar is tricky for a sentence like this, but here's how I'd phrase it if I understand correctly

```suggestion
    // Only ask for feedback if the user is not an admin or doesn't have write access (from a team membership)
```